doc/source/tutorial/stats/sampling_srou.rst

```diff
@@ -0,0 +1,90 @@
+.. _sampling-srou:
+
+Simple Ratio-of-Uniforms (SROU)
+===============================
+
+.. currentmodule:: scipy.stats.sampling
+
+* Required: PDF, area under PDF
```

Comment:
If the area is required it should be noted in the table. Or here it should say
```suggestion
* Required: PDF, area under PDF if different than 1
```

Reply:
done.